src/Content/Text/BBCode.php

```diff
@@ -436,15 +436,10 @@ private static function proxyUrl($image, $simplehtml = false)
 		}
 	}
 
-	public static function scaleExternalImages($srctext, $include_link = true, $scale_replace = false)
+	public static function scaleExternalImages($srctext)
 	{
-		// Suppress "view full size"
-		if (intval(Config::get('system', 'no_view_full_size'))) {
-			$include_link = false;
-		}
-
 		// Picture addresses can contain special characters
```

Comment:
I guess this comment is now superfluous, isn't it?

Reply:
Yup.